Userland/Libraries/LibWeb/Streams/AbstractOperations.cpp

```diff
@@ -435,6 +435,21 @@ void readable_stream_default_reader_error_read_requests(ReadableStreamDefaultRea
     }
 }
 
+// https://streams.spec.whatwg.org/#abstract-opdef-readablestreambyobreadererrorreadintorequests
+void readable_stream_byob_reader_error_read_into_requests(ReadableStreamBYOBReader& reader, JS::Value error)
+{
+    // 1. Let readIntoRequests be reader.[[readIntoRequests]].
+    // 2. Set reader.[[readIntoRequests]] to a new empty list.
+    auto read_requests = move(reader.read_into_requests());
```

Comment:
I see that this is copied from the above for the default reader, but ... Does this *really* set the reader's readIntoRequests slot into an empty list? I think it would be more convincing if this was either a member function on the reader and assigned the slot to be `{}`, or if there was an explicit set_read_into_requests() call.

Reply:
Yeah, fair point. I was just following the above for consitency, as you point out and figured it may be a pattern we may jjst have, but you're right - it does seem a bit weird, even if it does work.

I think its worth changing this, and the above call as well, will do 😀